### ISSUE TITLE
Scylla container image stopped logging to stdout

### ISSUE THREAD

**Issue by zimnx** [scylladb/scylla#10270]
In Scylla 4.6.0, default arguments supplied by container image changed the behavior of Scylla to stop logging to stdout. 

in 4.5.x:
```
root@simple-cluster-us-east-1-us-east-1a-0:/# grep -nri "log-to-stdout" /etc/
/etc/sysconfig/scylla-server:38:SCYLLA_ARGS="--log-to-syslog 0 --log-to-stdout 1 --default-log-level info --network-stack posix"
```
where in 4.6.0:
```
root@scyllacluster-5-eu-west-1-eu-west-1b-2:/# grep -nri "log-to-stdout" /etc/
/etc/default/scylla-server:41:SCYLLA_ARGS="--log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix"
```

This breaks Operator deployments, because after version upgrade, users are not able to read Scylla logs.

*Installation details*
Scylla version (or git commit hash): 4.6.0
OS (RHEL/CentOS/Ubuntu/AWS AMI): Container image

**Comment by nyh** [comment]
Wow... In `dist/docker/etc/sysconfig/scylla-server` we have:
```
SCYLLA_ARGS="--log-to-syslog 0 --log-to-stdout 1 --default-log-level info --network-stack posix"
```

which is what you wanted. BUT, we also have `dist/common/sysconfig/scylla-server`, and this one has the opposite order:
```
SCYLLA_ARGS="--log-to-syslog 1 --log-to-stdout 0 --default-log-level info --network-stack posix"
```

The curious thing is that both files and conflicting orders existed since 2018, added by @syuu1228.
I suspect we copy both files over another another, and for some reason the order changed in 4.6.0 and we get the wrong order (the "common" version overwrote the "docker" version).

@syuu1228 do we really need two copies of the same file? Why did the "winning" file change recently? How can we fix this?

**Comment by syuu1228** [comment]
Seems like we broke this since we switched to Ubuntu image (aa7c40ba50f0132c8a83a40936ec08917fb9c527).
We mistakenly copy dist/docker/etc/sysconfig/scylla-server to /etc/sysconfig/scylla-server, which is not used in Ubuntu (it should be /etc/default/scylla-server).
So /etc/default/scylla-server is just default configuration of scylla-server .deb package, --log-to-stdout is 0, same as normal installation.

Not directly related this bug, but I think @nyh commend is correct.
We probably should drop this duplicated file, configure /etc/default/scylla-server in build_docker.sh instead.
